subsys/bluetooth/host/gatt.c

```diff
@@ -4651,20 +4651,19 @@ int bt_gatt_unsubscribe(struct bt_conn *conn,
 void bt_gatt_cancel(struct bt_conn *conn, void *params)
 {
 	struct bt_att_req *req;
-	bt_att_func_t func;
-
+	bt_att_func_t func = NULL;
+	
```

Comment:
Looks like you accidentally added a tab here :) 

Suggest to set your editor to remove all trailing whitespace.